testnets/sgenetworktestnet/assetlist.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_id": "sge-network-3",
```

Comment:
please replace chain_id with chain_name (as sgenetworktestnet)

Reply:
Our chain id is sge-network-3, shall we keep the folder name as sge-network-3 ?

Reply:
@JeremyParish69 Got your point, will do that. cc @shang-chi-7

Reply:
no, the folder name sgenetworktestnet or saagetestnet would be most appropriate. The chain_id only goes in the chain.json file as `chain_id`.